### PR TITLE
Teamcity intermittent test failures fix

### PR DESCRIPTION
**Pull request by labkey-sweta**
#### Rationale
Fixing two intermittent test failures 

1. TargetedMSUtilizationCalendarTest.testUtilizationCalendarActions
2. TargetedMSSampleManagerIntegrationTest

#### Related Pull Requests
https://github.com/LabKey/testAutomation/pull/2021

#### Changes
*

### CODE REVIEW

File: test/src/org/labkey/test/components/targetedms/UtilizationCalendarWebPart.java [hunk 1]
```diff
@@ -73,7 +73,7 @@ public UtilizationCalendarWebPart markOffline(String startDate, @Nullable String
         ModalDialog modalDialog = clickDate(startDate);
         getWrapper().setFormElement(Locator.name("event-description").findElement(modalDialog.getComponentElement()), description);
         getWrapper().setFormElement(Locator.name("event-end-date").findElement(modalDialog.getComponentElement()), endDate);
-        modalDialog.dismiss("Save", 0);
+        doAndWaitForElementToRefresh(() -> modalDialog.dismiss("Save", 0), elementCache().dayLoc.withText(extractDay(startDate)), 10);
```
Comment (labkey-tchad) [inline test/src/org/labkey/test/components/targetedms/UtilizationCalendarWebPart.java]: FYI, none of the calls to `modalDialog.dismiss` in this class actually wait for the dialog to go away. Making them do so might improve its general reliability.

```suggestion
        doAndWaitForElementToRefresh(() -> modalDialog.dismiss("Save"), elementCache().dayLoc.withText(extractDay(startDate)), 10);
```

File: test/src/org/labkey/test/tests/panoramapremium/TargetedMSSampleManagerIntegrationTest.java [hunk 2]
```diff
@@ -114,10 +115,7 @@ public void testSampleTypeNavigation()
     private void createSampleType(String sampleName)
     {
         projectMenu().navigateToFolder(getProjectName(), Sample_Manager_Subfolder);
-        SampleTypeHelper sampleTypeHelper = new SampleTypeHelper(this);
-        SampleTypeDefinition sampleTypeDefinition = new SampleTypeDefinition(sampleName);
-        sampleTypeHelper.createSampleType(sampleTypeDefinition);
-        sampleTypeHelper.goToSampleType(sampleName);
+        SampleTypeAPIHelper.createEmptySampleType(getProjectName(), new SampleTypeDefinition(sampleName));
```
Comment (labkey-tchad) [inline test/src/org/labkey/test/tests/panoramapremium/TargetedMSSampleManagerIntegrationTest.java]: This doesn't look like it's creating the sample type in the correct location
```suggestion
        SampleTypeAPIHelper.createEmptySampleType(getProjectName() + Sample_Manager_Subfolder, new SampleTypeDefinition(sampleName));
```

**Review by labkey-tchad — CHANGES_REQUESTED** [review]

**Review by labkey-danield — APPROVED** [review]

**Review by labkey-danield — CHANGES_REQUESTED** [review]
The pathway used to create the sample type is not correct.
Please have a successful run on TeamCity before asking for a code review.

**Review by labkey-tchad — APPROVED** [review]

File: test/src/org/labkey/test/components/targetedms/UtilizationCalendarWebPart.java [hunk 3]
```diff
@@ -73,7 +73,7 @@ public UtilizationCalendarWebPart markOffline(String startDate, @Nullable String
         ModalDialog modalDialog = clickDate(startDate);
         getWrapper().setFormElement(Locator.name("event-description").findElement(modalDialog.getComponentElement()), description);
         getWrapper().setFormElement(Locator.name("event-end-date").findElement(modalDialog.getComponentElement()), endDate);
-        modalDialog.dismiss("Save", 0);
+        doAndWaitForElementToRefresh(() -> modalDialog.dismiss("Save"), elementCache().dayLoc.withText(extractDay(startDate)), 10);
```
Comment (labkey-tchad) [inline test/src/org/labkey/test/components/targetedms/UtilizationCalendarWebPart.java:76]: With the change in testAutomation, you should be able to remove the `, 0` from all of the calls to `modalDialog.dismiss`

**Review by labkey-danield — APPROVED** [review]

**Pull request merged**